src/main/java/edu/kis/powp/jobs2d/command/ICanvas.java

```diff
@@ -0,0 +1,6 @@
+package edu.kis.powp.jobs2d.command;
+
+public interface ICanvas {
+	 int getWidth();
+	 int getHeight();
```

Comment:
Please, do not assume that each canvas wound be rectangular (you can change name to IRectangularCanvas)